documentation/book/assembly-tls-sidecar.adoc

```diff
@@ -10,17 +10,16 @@
 
 = TLS sidecar
 
-Sidecar is a container which is running in a pod and serves an auxiliary purpose.
+A sidecar is a container which is running in a pod and serves an auxiliary purpose.
 The purpose of the TLS sidecar is to encrypt or decrypt the communication between {ProductName} components and Zookeeper since Zookeeper does not support TLS encryption natively.
```

Comment:
In {ProductName}, the TLS sidecar uses TLS to encrypt and decrypt all communication between the various components and Zookeeper. Zookeeper does not have native TLS support.